File: src/main/java/io/github/alexiscomete/lapinousecond/worlds/WorldWithCoos.java
package io.github.alexiscomete.lapinousecond.worlds;

public class WorldWithCoos extends World {

    public WorldWithCoos(int travelPrice, String name, String nameRP, String progName, String desc) {
        super(travelPrice, name, nameRP, progName, desc);
    }

    @Override
    public int getPriceForDistance(int distance, boolean place) {
        return 0;
    }

    @Override
    public int getDistance(Place place1, Place place2) {
        return 0;
    }
}

File: src/main/java/io/github/alexiscomete/lapinousecond/worlds/WorldWithCoos.kt
package io.github.alexiscomete.lapinousecond.worlds

class WorldWithCoos(travelPrice: Int, name: String?, nameRP: String?, progName: String?, desc: String?) :
    World(travelPrice, name!!, nameRP!!, progName!!, desc!!) {
    override fun getPriceForDistance(distance: Double, place: Boolean): Double {
        return 0.0
    }

    override fun getDistance(place1: Place?, place2: Place?): Double {
        return 0.0
    }
}